parse-package-set/src/Main.purs

```diff
@@ -58,7 +58,7 @@ main = runAff_ (either throwException mempty) do
 
   writeTextFile UTF8 (tmpPath <> "/spago.yaml") defaultSpagoYaml
 
-  let execOptsFn execSyncOptions = execSyncOptions { cwd = Just tmpPath }
+  let execOptsFn = _ { cwd = Just tmpPath }
```

Comment:
I see you updated this to a section, but maybe just inline it?

Reply:
fixed :-)